<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.3050" tests="3" failures="1">
  <testsuite name="Root Suite" timestamp="2023-03-29T12:29:50" tests="0" file="cypress/e2e/recommendations.cy.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="Series" timestamp="2023-03-29T12:29:50" tests="3" time="0.3050" failures="1">
    <testcase name="Series &quot;before each&quot; hook for &quot;Verify title and creator are shown&quot;" time="0.3050" classname="&quot;before each&quot; hook for &quot;Verify title and creator are shown&quot;">
      <failure message="The following error originated from your application code, not from Cypress.

  &gt; Object(...) is not a function

When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

This behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.

https://on.cypress.io/uncaught-exception-from-application

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Series`" type="TypeError"><![CDATA[TypeError: The following error originated from your application code, not from Cypress.

  > Object(...) is not a function

When Cypress detects uncaught errors originating from your application it will automatically fail the current test.

This behavior is configurable, and you can choose to turn this off by listening to the `uncaught:exception` event.

https://on.cypress.io/uncaught-exception-from-application

Because this error occurred during a `before each` hook we are skipping the remaining tests in the current suite: `Series`
    at WrappedRecommendationsSlider (http://localhost:4000/main.iframe.bundle.js:71038:80)
    at undecoratedStoryFn (http://localhost:4000/vendors~main.iframe.bundle.js:30221:38)
    at http://localhost:4000/vendors~main.iframe.bundle.js:20826:21
    at http://localhost:4000/vendors~main.iframe.bundle.js:30589:14
    at renderWithHooks (http://localhost:4000/vendors~main.iframe.bundle.js:124296:18)
    at mountIndeterminateComponent (http://localhost:4000/vendors~main.iframe.bundle.js:128060:13)
    at beginWork (http://localhost:4000/vendors~main.iframe.bundle.js:129573:16)
    at HTMLUnknownElement.callCallback (http://localhost:4000/vendors~main.iframe.bundle.js:112155:14)
    at Object.invokeGuardedCallbackDev (http://localhost:4000/vendors~main.iframe.bundle.js:112204:16)
    at invokeGuardedCallback (http://localhost:4000/vendors~main.iframe.bundle.js:112268:31)]]></failure>
    </testcase>
  </testsuite>
</testsuites>